packages/teams-js/src/public/authentication.ts

```diff
@@ -668,6 +668,34 @@ export namespace authentication {
      * Limited to Microsoft-internal use
      */
     ver: string;
+    /**
+     * @hidden
+     * Stores the residency of the user.
```

Comment:
>residency [](http://example.com/codeflow?start=18&length=9)

Maybe consider calling it "data residency" of the user, since that's the term used elsewhere?